docs/content/users/quickstart.md

```diff
@@ -250,8 +249,7 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
             mkdir my-statamic-app
             cd my-statamic-app
             ddev config --project-type=laravel --docroot=public --create-docroot
-            ddev composer create --prefer-dist --no-install --no-scripts statamic/statamic
-            ddev composer install
+            ddev composer create --prefer-dist statamic/statamic
```

Comment:
The only nitpick i would have is to change that line to `ddev composer create --prefer-dist statamic/statamic -y` that the composer create process could run through without any confirmation by the user.